components/raftstore-v2/src/fsm/peer.rs

```diff
@@ -1,37 +1,50 @@
 // Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
 
+//! This module contains the peer implementation for batch system.
+//!
+//! You may get confused about the peer in fsm and peer in raft module. The
```

Comment:
Another thing, could you comment when should we put implementations in `operation/xx`?